addons/mail/models/mail_message.py

```diff
@@ -414,8 +413,18 @@ def message_fetch_failed(self):
         return messages._format_mail_failures()
 
     @api.model
-    def message_fetch(self, domain, limit=20):
-        return self.search(domain, limit=limit).message_format()
+    def message_fetch(self, domain, moderated_channels=None, limit=20):
```

Comment:
Isn't it better to put the new moderated_channels argument after limit so that a call with positional arguments still work ?

Side note: I would use moderated_channel_ids to note this is an id list, not a record set.